dashboard/test/controllers/admin_search_controller_test.rb

```diff
@@ -17,6 +17,40 @@ class AdminSearchControllerTest < ActionController::TestCase
   generate_admin_only_tests_for :find_students
   generate_admin_only_tests_for :lookup_section
 
+  #
+  # find_student tests
+  #
+
+  test 'find_students flashes warning if multiple teachers match' do
+    create_list :teacher, 2, name: 'multiple'
+
+    get :find_students, params: {teacherNameFilter: 'multiple'}
+
+    assert_response :success
+    assert_select '.container .alert-danger', 'Multiple teachers matched the name and email search criteria.'
```

Comment:
Intersting also that it's (presumably) working for the `undelete_section` tests but not for the `lookup_section` test